ui/pages/_app.js

```diff
@@ -594,8 +594,12 @@ class MesheryApp extends App {
                         ) : (
                           <>
                             {' '}
-                            Built with <FavoriteIcon className={classes.footerIcon} /> by the Layer5
-                            Community
+                            Built with{' '}
+                            <FavoriteIcon
+                              style={{ color: '#00b39f' }}
```

Comment:
```suggestion
                              style={{ color: this.state.theme === 'dark' ? theme.palette.secondary.focused : '' }}
```

@VivekBaldha0712 this is what requested, please test this change if works update the PR

Reply:
It would be helpful if you can add screen shot after the changes

Reply:
@VivekBaldha0712 ... a screenshot of the UI, not the code.

Reply:
Sure. Here it is...
![image](https://github.com/meshery/meshery/assets/86009821/4182eb3d-a338-47fb-9f63-d8fb9fa47e16)

Reply:
Thankyou for adding the screenshot please make sure that we only change color to `#00b396` in dark mode only